Makefile

```diff
@@ -21,21 +21,22 @@ LOCAL_GID := $(shell id -g)
 CUID ?= $(LOCAL_UID)
 CGID ?= $(LOCAL_GID)
 
+# Define current directory only once
+CURDIR=$(shell pwd)
+
 # Define network name.
 COMPOSE_NET_NAME := $(COMPOSE_PROJECT_NAME)_front
 
 SDC_SERVICES=$(shell docker-compose config --services)
 # Determine database data directory if defined
-DB_MOUNT_DIR=$(shell cd docker && realpath $(DB_DATA_DIR))/
+DB_MOUNT_DIR=$(shell echo $(CURDIR)/$(shell basename $(DB_DATA_DIR)))
 ifeq ($(findstring mysql,$(SDC_SERVICES)),mysql)
```

Comment:
This curdir should be related to docker compose

Reply:
I double checked results, both commands `cd docker && realpath $(DB_DATA_DIR))/` and `$(CURDIR)/$(shell basename $(DB_DATA_DIR))` return exact same string. So this way looks ok